### PR TITLE
Add '/routing bgp template' path

### PR DESCRIPTION
**Pull request by derdeagle**
##### SUMMARY
Add the path "/routing bgp template".

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- api_info
- api_modify

##### ADDITIONAL INFORMATION
None.

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Thanks again!

File: changelogs/fragments/243-add-routing-bgp-template-path.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - add missing path "/routing bgp template" (https://github.com/ansible-collections/community.routeros/pull/243).
```
[the remark this answers is not in the document]
Comment (derdeagle) [reply changelogs/fragments/243-add-routing-bgp-template-path.yml]: Yep, way better now - thank you!

**Comment by derdeagle** [comment]
> Thanks again!

Always a pleasure ;)

**Comment by felixfontein** [comment]
Thanks!

**Pull request merged**